packages/inquirer/lib/prompts/checkbox.js

```diff
@@ -2,9 +2,9 @@
  * `list` type prompt
  */
 
-import chalk from 'chalk';
-import cliCursor from 'cli-cursor';
-import figures from 'figures';
+import chalk from '#chalk';
```

Comment:
Is there eslint rules to enforce to use the `#` imports if the dependency is found in the package.json import map. That would help enforce it for future collaborators.

Reply:
We'll also need to fix the `node/no-missing-import` eslint error before merging.